<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="outViewLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>NICOS devices</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filter"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="tree">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>0</number>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/jump</normaloff>:/jump</iconset>
   </property>
   <property name="text">
    <string>Move...</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionFix">
   <property name="text">
    <string>Fix...</string>
   </property>
  </action>
  <action name="actionRelease">
   <property name="text">
    <string>Release</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/emergency</normaloff>:/emergency</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPlotHistory">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/find</normaloff>:/find</iconset>
   </property>
   <property name="text">
    <string>Plot history...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
